lint.sh

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env bash
-#
-# Lint all modules with golangci-lint.
-#
-#   lint.sh - lints all modules in the repository root, or the current folder if
-#   not in a git workspace.
-#
-#   lint.sh [folder] - lints all modules in the specified folder.
-
-REPO=`git rev-parse --show-toplevel 2> /dev/null`
-if [[ $? != 0 ]]; then
-    REPO=`pwd`
-fi
-
-# Root is the input argument, or REPO if no input argument.
-ROOT=${1:-$REPO}
-echo "Linting all modules under" $ROOT
-
-set -e
-
-shopt -s expand_aliases
-
-export GO111MODULE=on
-
-go version
-
-pushd $ROOT > /dev/null
-
-ROOTPATH=$(go list -m -f {{.Dir}} 2>/dev/null)
-ROOTPATHPATTERN=$(echo $ROOTPATH | sed 's/\\/\\\\/g' | sed 's/\//\\\//g')
-MODPATHS=$(go list -m -f {{.Dir}} all 2>/dev/null | grep "^$ROOTPATHPATTERN")
```

Comment:
Are these three vars computed right?  I though there were some changes to these in dcrd and dcrwallet to fix something, but it never made it to dcrdata.